frontend/src/scenes/notebooks/Notebook/NotebookPopover.tsx

```diff
@@ -0,0 +1,122 @@
+import { useActions, useValues } from 'kea'
+import clsx from 'clsx'
+import './NotebookPopover.scss'
+import { Notebook } from './Notebook'
+import { notebookPopoverLogic } from 'scenes/notebooks/Notebook/notebookPopoverLogic'
+import { LemonButton } from '@posthog/lemon-ui'
+import { IconFullScreen, IconChevronRight, IconLink } from 'lib/lemon-ui/icons'
+import { useEffect, useRef } from 'react'
+import { useKeyboardHotkeys } from 'lib/hooks/useKeyboardHotkeys'
+import { NotebookListMini } from './NotebookListMini'
+import { notebooksListLogic } from './notebooksListLogic'
+import { NotebookExpandButton, NotebookSyncInfo } from './NotebookMeta'
+import { notebookLogic } from './notebookLogic'
+import { urls } from 'scenes/urls'
+
+export function NotebookPopover(): JSX.Element {
+    const { visibility, shownAtLeastOnce, fullScreen, selectedNotebook, initialAutofocus, dropListeners } =
+        useValues(notebookPopoverLogic)
+    const { setVisibility, setFullScreen, selectNotebook, setElementRef } = useActions(notebookPopoverLogic)
+    const { createNotebook } = useActions(notebooksListLogic)
+    const { notebook } = useValues(notebookLogic({ shortId: selectedNotebook }))
+
+    const ref = useRef<HTMLDivElement>(null)
+
+    // NOTE: This doesn't work for some reason, possibly due to the way the editor is rendered
```

Comment:
I'm not sure I understand this comment. This seems to close fullscreen for me

Reply:
Ah yeah I just implemented it wrong. Updated now to exit full screen and then hide it alltogether